pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -67,6 +69,10 @@ func NewManager(
 	recorder record.EventRecorder,
 	nodeRef *api.ObjectReference,
 	clock util.Clock) (Manager, lifecycle.PodAdmitHandler, error) {
+
+	// TODO: pass this into the manager
+	withImageFs := false
```

Comment:
actually, i fixed this up, it cant be passed in NewManager because knowing if imagefs and rootfs are unified requires knowledge that is runtime dependent.  i am going to build the rank functions in the Start() and just start the manager after we start cAdvisor.